fl4health/clients/basic_client.py

```diff
@@ -187,8 +362,16 @@ def train_step(self, input: torch.Tensor, target: torch.Tensor) -> Tuple[Losses,
 
     def val_step(self, input: torch.Tensor, target: torch.Tensor) -> Tuple[Losses, Dict[str, torch.Tensor]]:
         """
-        Given input and target, compute loss, update loss and metrics
+        Given input and target, compute loss, update loss and metrics.
```

Comment:
In the doc string, perhaps mention that this function is meant to execute evaluation on a single batch?